### ISSUE TITLE
Bug Summary

### ISSUE THREAD

**Issue by GiammaFer75** [wilhelm-lab/oktoberfest#73]
**Describe the bug**
Sorry but I am quite new about Docker and I did not understand why in the Dockerfile it is called a file that apparently is missing (maybe must be generated by the previous dockerfile commands????). 
As a consequence, I got this error while I am trying to build the Docker image:
```
ERROR: failed to solve: failed to compute cache key: failed to calculate checksum of ref 581922ad-c5e5-466c-9dac-9be6e15ce4bf::il5wmk5wwugil7djvxrpnqinn: "/hash.file": not found
```

**To Reproduce**

1. I downloaded the repository through git clone
2. From the oktoberfest folder I run: `sudo docker build -t oktoberfest .`

**System [please complete the following information]:**

-   OS: Ubuntu 22.04.2 LTS
-   Virtual environment: Conda

**Additional context**
The full output of the Docker build command before the error is :

```
[+] Building 1.6s (26/26) FINISHED                                                                                                                                              
 => [internal] load build definition from Dockerfile                                                                                                                       0.0s
 => => transferring dockerfile: 1.81kB                                                                                                                                     0.0s
 => [internal] load .dockerignore                                                                                                                                          0.0s
 => => transferring context: 50B                                                                                                                                           0.0s
 => [internal] load metadata for docker.io/library/python:3.8.12                                                                                                           1.4s
 => CANCELED [ 1/22] FROM docker.io/library/python:3.8.12@sha256:53cb5152064a7e7b485ad42704ea63c5155b264c82e7f17de99d3aa28e4f8956                                          0.0s
 => => resolve docker.io/library/python:3.8.12@sha256:53cb5152064a7e7b485ad42704ea63c5155b264c82e7f17de99d3aa28e4f8956                                                     0.0s
 => => sha256:60d1cda1542582095795c25bff869b0c615e2a913c4026ed0313ede156b60468 2.22kB / 2.22kB                                                                             0.0s
 => => sha256:53cb5152064a7e7b485ad42704ea63c5155b264c82e7f17de99d3aa28e4f8956 1.86kB / 1.86kB                                                                             0.0s
 => => sha256:52bb9574949f3991ed435968cbb3b68b7b3ab3d041633ad40866ee447896c176 8.59kB / 8.59kB                                                                             0.0s
 => [internal] load build context                                                                                                                                          0.0s
 => => transferring context: 344.69kB                                                                                                                                      0.0s
 => CACHED [ 2/22] RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF                                          0.0s
 => CACHED [ 3/22] RUN echo "deb https://download.mono-project.com/repo/debian stable-buster main" | tee /etc/apt/sources.list.d/mono-official-stable.list                 0.0s
 => CACHED [ 4/22] RUN apt-get update                                                                                                                                      0.0s
 => CACHED [ 5/22] RUN apt-get install -y         mono-devel         ssh         zip RUN rm -rf /var/lib/apt/lists/*                                                       0.0s
 => CACHED [ 6/22] WORKDIR /root                                                                                                                                           0.0s
 => CACHED [ 7/22] ADD keys /root/.ssh                                                                                                                                     0.0s
 => CACHED [ 8/22] RUN chmod 700 /root/.ssh/id_rsa                                                                                                                         0.0s
 => CACHED [ 9/22] RUN pip install poetry==1.3.2                                                                                                                           0.0s
 => CACHED [10/22] RUN poetry config virtualenvs.create false                                                                                                              0.0s
 => CACHED [11/22] ADD pyproject.toml /root/pyproject.toml                                                                                                                 0.0s
 => CACHED [12/22] ADD poetry.lock /root/poetry.lock                                                                                                                       0.0s
 => CACHED [13/22] RUN poetry install --no-root                                                                                                                            0.0s
 => CACHED [14/22] RUN ZIP=ubuntu.tar.gz                                                                                                                                   0.0s
 => CACHED [15/22] RUN wget https://github.com/percolator/percolator/releases/download/rel-3-05/$ZIP -O /tmp/$ZIP                                                          0.0s
 => CACHED [16/22] RUN tar xvzf /tmp/$ZIP                                                                                                                                  0.0s
 => CACHED [17/22] RUN chmod -R 755 /tmp/*                                                                                                                                 0.0s
 => CACHED [18/22] RUN dpkg -i percolator-v3-05-linux-amd64.deb                                                                                                            0.0s
 => CACHED [19/22] RUN rm /tmp/$ZIP                                                                                                                                        0.0s
 => CACHED [20/22] RUN rm -r /root/.ssh                                                                                                                                    0.0s
 => CACHED [21/22] ADD oktoberfest/ /root/oktoberfest                                                                                                                      0.0s
 => ERROR [22/22] ADD hash.file /root/hash.file                                                                                                                            0.0s
------
 > [22/22] ADD hash.file /root/hash.file:
------
Dockerfile:58
--------------------
  56 |     
  57 |     # Used by ProteomicsDB runs to describe the oktoberfest version
  58 | >>> ADD hash.file /root/hash.file
  59 |     
--------------------
ERROR: failed to solve: failed to compute cache key: failed to calculate checksum of ref 581922ad-c5e5-466c-9dac-9be6e15ce4bf::m26jfdzlh33j9vlfpl8kk62hu: "/hash.file": not found

```

**Comment by MatthewThe** [comment]
You can resolve this by running `git describe --long --dirty --always > hash.file` in the directory before building the image. 

@LLautenbacher do you remember why we don't run this command inside the Dockerfile?

**Comment by picciama** [comment]
This should be fixed in #77. I will merge into the development after merging #75.
